Framework/PythonInterface/mantid/utils/absorptioncorrutils.py

```diff
@@ -27,12 +33,137 @@ def _getBasename(filename):
     return name
 
 
-def calculate_absorption_correction(filename, abs_method, props, sample_formula, mass_density,
-                                    number_density=Property.EMPTY_DBL,
-                                    container_shape="PAC06",
-                                    num_wl_bins=1000,
-                                    element_size=1,
-                                    metaws=None):
+def _getCacheName(wkspname, cache_dir, abs_method):
+    """
+    Generate a MDF5 string based on given key properties.
+
+    :param wkspname: donor workspace containing absorption correction info
+    :param cache_dir: location to store the cached absorption correction
+    
+    return fileName(full path), sha1
+    """
+
+    # fix up the workspace name
+    prefix = wkspname.replace('__', '')
```

Comment:
There are no type hints for `wksname` and the docstring doesn't mention that `wkspname` is a string.

Reply:
`wkspname` = workspace_name, and name of a workspace in Mantid has to be a string :sweat_smile:

Reply:
I guessed as much, but I like code to be a bit more explicit. Not a biggie, anyway